mkinitcpio-install.sh

```diff
@@ -65,20 +65,23 @@ add_systemd_unit_X() {
     # and other unit files will be discovered and added.
     #   $1: path to rules file (or name of rules file)
 
-    local unit= rule= entry= key= value= binary= dep=
+    local unit_name= unit= rule= entry= key= value= binary= dep=
 
     # use simple unit name
-    unit=$(basename $1)
+    unit_name=$(basename $1)
     quiet "add systemd unit $unit"
     
     # search in all standard locations
-    unit=$(PATH=/etc/systemd/system:/usr/lib/systemd/system:/lib/systemd/system type -P "$unit")
+    unit=$(PATH=/etc/systemd/system:/usr/lib/systemd/system:/lib/systemd/system type -P "$unit_name")
     if [[ -z $unit ]]; then
-        # complain about not found unit file
+        error "unit not found: %s" "$unit_name"
         return 1
     fi
 
-    add_file "$unit"
+    if [[ ! -e "$BUILDROOT/$unit" ]]; then
```

Comment:
perhaps we want to overwrite - always? see:
https://git.archlinux.org/mkinitcpio.git/tree/functions#n510

Reply:
Yea, I saw that but didn't think it was neccessary since the output should never changes.
Will do it now.